main.py

```diff
@@ -1,60 +1,38 @@
 from eoscript import Exposure, Script
 
-#------------------------------------------------------------------------------
-# Write you scirpt!
-
-_1 = Exposure(1)
-
+# All times are UTC!!!
+# All times are UTC!!!
+# All times are UTC!!!
 script = Script(
```

Comment:
I believe we should not be writing the contact times in the file and let simulated second contact take care of it based on the GPS coordinates that we manually set.

This will avoid the situation if we forget to comment out these lines and use that file accidentally on eclipse day.

Reply:
Sure, but it's super useful while one is developing the script to know the durations between contact times.  I also want to use HH:MM:SS for the partials captures, and contact times are needed for that.  The produced script only uses relative offsets.